koku/api/report/test/aws/tests_queries.py

```diff
@@ -691,7 +696,8 @@ def test_execute_query_curr_month_by_avail_zone(self):
             month_data = data_item.get("azs")
             self.assertEqual(month_val, cmonth_str)
             self.assertIsInstance(month_data, list)
-            self.assertEqual(len(month_data), self.availability_zone_count_cur_mo)
+            # Add 1 to the count for "no-az"
+            self.assertEqual(len(month_data), self.availability_zone_count + 1)
```

Comment:
This test was just changed in https://github.com/project-koku/koku/pull/2332 to fix an intermittent test error. Want to make sure this isn't re-introducing that.

Reply:
I'll look into it. Thanks for catching it.

Reply:
I've re-aligned this with the newer changes from #2332